miio/integrations/vacuum/roborock/vacuum.py

```diff
@@ -746,6 +756,49 @@ def set_carpet_cleaning_mode(self, mode: CarpetCleaningMode):
             == "ok"
         )
 
+    @command()
+    def enable_dust_collection(self):
+        """Enable automatic dust collection."""
+        return (
+            self.send("set_dust_collection_switch_status", {"status": 1})[0]
+            == "ok"
+        )
+
+    @command()
+    def disable_dust_collection(self):
+        """Disable automatic dust collection."""
+        return (
+            self.send("set_dust_collection_switch_status", {"status": 0})[0]
+            == "ok"
+        )
```

Comment:
Please combine these to a single `set_dust_collection(self, enabled: bool)` method.